contracts/lines/AxelarLine.sol

```diff
@@ -9,9 +9,9 @@ import {AxelarExecutable} from "@axelar-network/axelar-gmp-sdk-solidity/contract
 import {IAxelarGateway} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol";
 import {IAxelarGasService} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol";
 import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/access/Ownable2Step.sol";
```

Comment:
Need in other lines?

Reply:
Yes, other lines also use `Ownable2Step`.

Reply:
It seems that lzApp doesn't use Ownable2Step https://github.com/LayerZero-Labs/solidity-examples/blob/main/contracts/lzApp/LzApp.sol#L5

Reply:
And CelerLine

Reply:
Yes, we use `Ownable2Step` include chain-id-mappings.
We could not change our upstream code except to implement Line in our way.